internal/controller/suite_test.go

```diff
@@ -109,6 +110,11 @@ var _ = BeforeSuite(func() {
 		UseExistingCluster: &useExistingCluster,
 	}
 
+	numaLogger := logger.New().WithName("controller-manager")
```

Comment:
why is this change?

Reply:
It didn't look like we were ever setting the log level in here previously, and if a test were to fail, it would be convenient to set the log level to 4 below and re-run.

Below, though, I see we are actually testing `configManager.LoadAllConfigs()` and could theoretically be actually creating a ConfigMap to read in down there with the log level in it. 

This file is really seeming more and more like not a unit test, so I imagine we will replace a lot of this stuff anyway. If you prefer for me to take this out, I can.

Reply:
got it, I'm good with keep it here, but why not set to 4 initially to show debug log?

Reply:
I'm good with that